### DISCUSSION TITLE
bypass option

### DISCUSSION THREAD

**Discussion by Lesnikovsok** [petrleocompel/hikaxpro_hacs#113] [Ideas]
hello,
is there any option to force arm in "stay" (bypass triggered sensors). or to get a popup zone x triggered. do you want to bypass it?
why is this needed? i have a house with 100+ sensors and in summer days some windows are opened.

**Comment by petrleocompel** [comment]
There is no bypass. Only thing that could you do is to set up in hikvision app that you want to exclude that sensor and it will not be checked during arming. 
There is no thing in API to help this. 

Or you could split everything in zones. And setup a automation which will try to arm every zone separately.

**Comment by Lesnikovsok** [comment]
Dont want to open new one. I think it might be related to previous question.
In HA i have auto arm in stay mode at night. This week wife forgot to close the main door (door contact final door exit, sensor is not bypassed). So after auto arm, it went straight to alarm. Is there any changes i can do, so we dont have incidents like this again.

**Reply by petrleocompel** [reply]
Well you did not describe the target solution.
But in your automation you could do something like "pre-arm check" if it is not okay just notify user or something. 

I do something similar as "preflight" for lot of my smart home. "easy example" "indoor covers" if window is open cover never can go down. only up. 

So before you arm just check prerequisite and if is satisfied you can arm.
